sdk/js/src/service/applications.js

```diff
@@ -44,6 +45,13 @@ class Applications {
     })
     this.Link = new Link(api.As)
     this.Devices = new Devices(api, { proxy, stackConfig })
+    this.Devices = new Devices(api, { proxy })
```

Comment:
I guess this missed a rebase here. The `stackConfig` needs to be passed to the Device.